Add description on how to use hide, collapsed (#9369)
microsoft/pxt: docs/writing-docs/snippets.md

```
````

### hide

When used in snippets that produce an image of blocks, the renderer will remove the code or block following the a
comment containing **@hide**. Use `// @hide` in blocks and TypeScript, and `# @hide` in Python.

This will most often be useful when hiding set up code - e.g. a variable or function declaration.

````
```blocks
// @hide
function myCoolFunction() {

}

// @hide
let mySprite = sprites.create(img`1`);

mySprite.x += 50;
myCoolFunction();
```
````

### collapsed

When used in snippets, the following function or event will show up collapsed in the hint. This
can also be used in template code in order to have code start off collapsed in the user's workspace,

> in case there is code that should exist but not be focused on immediately / that you will call out a few steps in.

Reviewer: nit:

```suggestion
in case there is code that should exist but not be focused on immediately, or that you will call out in subsequent steps.
```
